llvm/test/tools/sycl-post-link/device-requirements/device-requirements.ll

```diff
@@ -0,0 +1,207 @@
+; Original code:
+; #include <sycl/sycl.hpp>
+; [[__sycl_detail__::__uses_aspects__(sycl::aspect::fp64, sycl::aspect::cpu)]] void foo() {}
+; [[__sycl_detail__::__uses_aspects__(sycl::aspect::queue_profiling, sycl::aspect::cpu, sycl::aspect::image)]] void bar() {
+;   foo();
+; }
+; int main() {
+;   sycl::queue q;
+;   q.submit([&](sycl::handler &cgh) {
+;     cgh.single_task([=]() { bar(); });
+;   });
+; }
+
+; RUN: sycl-post-link -split=auto -o %s -o %t.files.table
```

Comment:
We should have a variation of this test with per-kernel split to ensure that we generate properties correctly for each resulting module